Source/Android/app/build.gradle

```diff
@@ -89,3 +92,7 @@ dependencies {
     // Allows FRP-style asynchronous operations in Android.
     compile 'io.reactivex:rxandroid:1.2.1'
 }
+
+def getVersion() {
+    return 'git describe --abbrev=0'.execute([], project.rootDir).text.trim()
```

Comment:
https://github.com/dolphin-emu/dolphin/blob/master/CMakeLists.txt#L133 is used for all other systems.